source/Introduction/About-SpaceROS.rst

```diff
@@ -1,12 +1,96 @@
 About Space ROS
```

Comment:
The filename of this article should probably be `About-Space-ROS.rst` to conform with the official spelling `Space ROS` rather than `SpaceROS`.

Reply:
Agreed. Fixed.

I also renamed How-SpaceROS-Differs.rst to become How-Space-ROS-Differs.rst.